pyccl/core.py

```diff
@@ -35,7 +36,11 @@
 
 # List which transfer functions can be used with the muSigma_MG
 # parameterisation of modified gravity
-valid_muSig_transfers = {'boltzmann_class', 'class'}
+
+# MI pending question
+# Matt: is this still used? I changed it anyway but don't see it being used
+# valid_muSig_transfers = {'boltzmann_class', 'class'}
+valid_muSig_transfers = {'boltzmann_isitgr', 'isitgr'}
```

Comment:
I am not sure is this line is actually needed. Just check but I might be wrong.

Reply:
Ah, on more. These are not needed so all removed.